TRANSLATION.md

```diff
@@ -5,22 +5,25 @@
 
 Check [existing translation pull-requests](https://github.com/letsencrypt/website/pulls?q=is%3Apr+is%3Aopen+label%3Atranslation) to see if someone else is already working on it.
 
+## Translate new pages, update or improve translation
+
+You can help with the translation work on https://crowdin.com/project/lets-encrypt-website
+
 ## How to add a new language
 
+If you are interested to add a new language, please open an issue with the information requested in:
+- `languageName`, `languageCode`, `beforeColon`, `description` and `numberFormat` in https://github.com/letsencrypt/website/blob/master/config/_default/languages.en.toml
+- `name` entries in https://github.com/letsencrypt/website/blob/master/config/_default/menu.en.toml
+
+
+Someone from Let's Encrypt will add the language to Crowdin so you can start translating on https://crowdin.com/project/lets-encrypt-website
+
+When enought pages are translated, someone from Let's Encrypt will create the first pull request:
```

Comment:
There is a small typo here: replace "enought" with "enough"

Reply:
Thank you! It's fixed